src/main/java/net/mcreator/blockly/data/ExternalBlockLoader.java

```diff
@@ -206,6 +209,52 @@ public void loadBlocksAndCategoriesInPanel(BlocklyPanel pane, ToolboxType toolbo
 		String toolbox_xml = FileIO
 				.readResourceToString("/blockly/toolbox_" + toolboxType.name().toLowerCase(Locale.ENGLISH) + ".xml");
 
+		//We add variable related blocks inside their category

```

Comment:
This section should be cached somehow, also consider moving this code to a new class or more appropriate one and just call it from here so we do not bulk this class